package.json

```diff
@@ -24,20 +26,38 @@
         "node": ">=10.0.0",
         "npm": ">=6.0.0"
     },
+    "dependencies": {
+        "@emotion/core": "^10.0.7",
+        "@emotion/styled": "^10.0.7",
+        "react": "^16.8.1",
+        "react-dom": "^16.8.1"
+    },
     "devDependencies": {
+        "@babel/core": "^7.2.2",
+        "@babel/plugin-proposal-class-properties": "^7.3.0",
+        "@babel/preset-env": "^7.3.1",
+        "@babel/preset-flow": "^7.0.0",
+        "@babel/preset-react": "^7.0.0",
+        "@emotion/babel-preset-css-prop": "^10.0.7",
+        "babel-loader": "^8.0.5",
+        "babel-plugin-emotion": "^10.0.7",
         "clean-css-cli": "4.2.1",
         "eslint": "5.1.0",
         "eslint-plugin-jest": "22.1.2",
```

Comment:
I believe we may need a couple of additional `eslint` plugins, see:
https://github.com/schalkneethling/mdn-feature-banner-component/blob/master/package.json#L40

Reply:
Thanks for that link! I got flow working for this PR, but forgot about eslint. I'll add the extra plugins.